opm/input/eclipse/EclipseState/Grid/EclipseGrid.cpp

```diff
@@ -2176,3 +2350,23 @@ std::vector<double> EclipseGrid::createDVector(const std::array<int,3>& dims, st
 
 
 
+namespace Opm {
+    EclipseGridLGR::EclipseGridLGR(std::string self_label, std::string father_label_, 
+                                   int father_lgr_level, std::size_t nx, std::size_t ny, 
+                                   std::size_t nz, vec_size_t father_lgr_index)
+    : EclipseGrid(nx,ny,nz), father_label(father_label_), father_global(father_lgr_index)
+    {
+        init_father_global();
+        lgr_label= self_label;
+        lgr_level = father_lgr_level + 1 ;   
```

Comment:
This might not be a problem for now, but... Dune's Grid interface follows this scheme where children of a cell ive in "parent cell level +1". However, currently, for CpGrid, "LGR1" is stored in level 1, "LGR2" stored in level 2 and so on... even though all parent cells belong to level zero.